### PR TITLE
Updates to support AWS EB as at 2014-05-16

### PR DESCRIPTION
**Pull request by jufemaiz**
This is confirmed as working, thought there may be an argument for using the environment variable $EB_CONFIG_APP_SUPPORT for /var/app/containerfiles ?

### CODE REVIEW

**Comment by 41studi0** [comment]
can you take a look this pull request @dignoe  ?

**Comment by jufemaiz** [comment]
https://github.com/dignoe/whenever-elasticbeanstalk/issues/7
https://github.com/dignoe/whenever-elasticbeanstalk/issues/8

**Comment by lime** [comment]
I would definitely use `$EB_CONFIG_APP_SUPPORT` in place of `/var/app/containerfiles`, since it is more likely to be future-proof.

Does `$EB_CONFIG_APP_SUPPORT` exist in the old environments that use `/var/app/support`? Because that could mean we wouldn't have to sacrifice backwards compatibility at least on that front, which would be great.

File: README.md [hunk 1]
```diff
@@ -99,7 +89,7 @@ end
 To run a task on all instance, omit the `roles` option.
 ```ruby
 every 1.minute do
-	command "touch /opt/elasticbeanstalk/support/.cron_check"
+	command "touch /opt/elasticbeanstalk/containerfiles/.cron_check"
```
Comment (lime) [inline README.md:92]: Also, could it be that this directory is available as an environment variable as well?

Comment (jufemaiz) [inline README.md:92]: Unfortunately that directory isn't in an ENV Variable.

```
AWS_ACCESS_KEY_ID =>
EB_CONFIG_SYSTEM_AWSEBAGENTID =>
PARAM1 =>
GEM_PATH =>
_ORIGINAL_GEM_PATH =>
AWS_SECRET_KEY =>
PARAM3 =>
EB_CONFIG_SYSTEM_AWSEBREFERRERID =>
PARAM4 =>
PARAM5 =>
PARAM2 =>
LESS_TERMCAP_mb =>
LESS_TERMCAP_md =>
LESS_TERMCAP_us =>
LESS_TERMCAP_se =>
LESS_TERMCAP_me =>
LESS_TERMCAP_ue =>
RUBYOPT => -I/usr/local/share/ruby/gems/2.0/gems/bundler-1.3.5/lib -rbundler/setup
SHELL => /bin/bash
SSH_TTY => /dev/pts/0
OLDPWD => /home/ec2-user
HOME => /home/ec2-user
AWS_PATH => /opt/aws
EC2_AMITOOL_HOME => /opt/aws/amitools/ec2
AWS_AUTO_SCALING_HOME => /opt/aws/apitools/as
EC2_HOME => /opt/aws/apitools/ec2
AWS_ELB_HOME => /opt/aws/apitools/elb
AWS_IAM_HOME => /opt/aws/apitools/iam
AWS_CLOUDWATCH_HOME => /opt/aws/apitools/mon
AWS_RDS_HOME => /opt/aws/apitools/rds
EB_ROOT => /opt/elasticbeanstalk
EB_CONFIG_SOURCE_BUNDLE => /opt/elasticbeanstalk/deploy/appsource/source_bundle
EB_CONFIG_FILE => /opt/elasticbeanstalk/deploy/configuration/containerconfiguration
JAVA_HOME => /usr/lib/jvm/jre
_ => /usr/local/bin/bundle
BUNDLE_BIN_PATH => /usr/local/share/ruby/gems/2.0/gems/bundler-1.3.5/bin/bundle
EB_CONFIG_APP_BASE => /var/app
EB_CONFIG_APP_SUPPORT => /var/app/containerfiles
EB_CONFIG_APP_ASSETS => /var/app/containerfiles/assets
EB_CONFIG_APP_LOGS => /var/app/containerfiles/logs
EB_CONFIG_APP_PIDS => /var/app/containerfiles/pids
EB_CONFIG_APP_CURRENT => /var/app/current
PWD => /var/app/current
BUNDLE_GEMFILE => /var/app/current/Gemfile
GEM_HOME => /var/app/current/vendor/bundle/ruby/2.0
PATH => /var/app/current/vendor/bundle/ruby/2.0/bin:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/opt/aws/bin:/home/ec2-user/bin
EB_CONFIG_APP_ONDECK => /var/app/ondeck
MAIL => /var/spool/mail/ec2-user
SHLVL => 1
HISTSIZE => 1000
SSH_CONNECTION => 121.44.135.239 54221 172.31.10.40 22
SSH_CLIENT => 121.44.135.239 54221 22
RUBY_VERSION => 2.00
COLUMNS => 204
LINES => 48
EB_CONFIG_HTTP_PORT => 80
USER => ec2-user
LOGNAME => ec2-user
LANG => en_AU.UTF-8
S3_BUCKET_NAME => energylink-staging
EB_CONFIG_SYSTEM_LOGPUBLICATIONCONTROL => false
RAILS_SKIP_MIGRATIONS => false
RAILS_SKIP_ASSET_COMPILATION => false
HISTCONTROL => ignoredups
LS_COLORS => rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=01;05;37;41:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.Z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.jpg=01;35:*.jpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.axv=01;35:*.anx=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=01;36:*.au=01;36:*.flac=01;36:*.mid=01;36:*.midi=01;36:*.mka=01;36:*.mp3=01;36:*.mpc=01;36:*.ogg=01;36:*.ra=01;36:*.wav=01;36:*.axa=01;36:*.oga=01;36:*.spx=01;36:*.xspf=01;36:
BUNDLE_WITHOUT => test:development
EB_CONFIG_APP_USER => webapp
TERM => xterm
LESSOPEN => |/usr/bin/lesspipe.sh %s
```

That's the ENV variables available. Could do some munging with the $EB_CONFIG_APP_SUPPORT though?

Comment (lime) [inline README.md:92]: Yeah, something like `touch $EB_ROOT/$(basename $EB_CONFIG_APP_SUPPORT)/.cron_check` should do it. Not sure if that's a good idea though.

Comment (lime) [inline README.md:92]: Oh wait, scratch that, I just realized this is just in the README as an example. We can expect users to know what directories they have available in their environment, so it doesn't really matter what directory is referred to here. :)

**Pull request closed without merging**